frontend/src/app.js

```diff
@@ -20,11 +20,24 @@ import share from './pages/share.js';
 es6Promise.polyfill();
 Raven.config(config.ravenPublicDSN).install();
 
-error();
-experiment();
-experiments();
-home();
-legacy();
-onboarding();
-retire();
-share();
+const routes = {
```

Comment:
Indeed. I would like to go back to the way it was before with one bundle per page so I don't have to reimplement client side routing, but only if I can solve the bundle bloat problem, and it is not a high priority.